src/Chapter04.Tests/Listing04.34.Tests.cs

```diff
@@ -6,13 +6,33 @@ namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter04.Listing04_34.Tests
     public class ProgramTests
     {
         [TestMethod]
-        public void Main_LogicalNotOnFalseBool_WriteTrue()
+        public void Main_Enter25ForHourOfDay_AndConditionFails() // fulfills left side of AND but not right
         {
             const string expected =
-                @"result = True";
+                @"";
 
             IntelliTect.TestTools.Console.ConsoleAssert.Expect(
-                expected, Program.Main);
+                expected, () => Program.Main("25"));
+        }
+
+        [TestMethod]
+        public void Main_Enter5ForHourOfDay_AndConditionFails() // fulfills right side of AND but not left
```

Comment:
I would not put the comment at the end of the line. This can cause formatting problems with wrapping. It is better to put the comment above the method or at the first line of the method.

Reply:
Hmm... suggestions from on existing+ test code rather than code changed could add up to a lot of items.